src/app/donut-chart/donut-chart.component.css

```diff
@@ -0,0 +1,24 @@
+svg {
+  height: 12rem;
+  width: 12rem;
+}
+
+.donut-percent {
+  font-size: 0.3rem;
```

Comment:
1 pixel jest najmniejszą widzialną wartością na ekranie. Nie powinniśmy "dzielić atomu". Takie operacje mogą również spowodować niechciane zblurowanie elementu na który nanosimy taką wartość. 
0.3rem to 4.8px, dlatego powinno się zaokrąglić wartość w pixelach do 5 i podzielić przez 16 (jeśli mamy na taką wartość domyślnie ustawioną czcionkę w aplikacji). Podsumowując font-size powinien wynosić 0.3125rem.
PS Tipem ułatwiającym życie jest używanie wartości 'co ćwiartkę' w remach czyli .25rem, .5rem, .75rem, 1rem, 1.25rem itd. A jeśli potrzebujemy wartości bardziej precyzyjnej to dopiero wtedy przeliczać :)

Reply:
Ma to sens :D Pozmieniałem